app/src/main/res/values-in/strings.xml

```diff
@@ -0,0 +1,39 @@
+<resources>
+    <string name="app_name">MVVM Boilerplate</string>
+    <string name="action_ok">OK</string>
+    <string name="action_cancel">BATALKAN</string>
+    <string name="message_user_unautorizhed">Anda telah login pada device lain</string>
+    <string name="msg_unexpected_error">Permintaan tidak dapat diproses, Silahkan coba lagi nanti</string>
+    <string name="label_login">Masuk</string>
+    <string name="label_email">Email</string>
+    <string name="label_password">Password</string>
+    <string name="label_forgot_password">Lupa kata sandi?</string>
```

Comment:
kata sandi or Password, try to be consistent

Reply:
are those words still necessary?
@fajarnuha

Reply:
will removed yah